components/contact/ContactForm.tsx

```diff
@@ -1,4 +1,36 @@
 // code
+<<<<<<< HEAD
```

Comment:
This is from git conflicts, your local code should have error, please remove this line and try again